### PR TITLE
Updated renovate.json to add correct commitMessagePrefix

### PR DESCRIPTION
**Pull request by DineshKumarRA**
### **User description**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Updated renovate.json to add correct commitMessagePrefix based on packages. Have also increased the prConcurrentLimit to 15 as I think 5 is too low for a big project. But happy to change this back.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

___

### **PR Type**
enhancement, configuration changes

___

### **Description**
- Added a new CI workflow specifically for renovate branches, including steps for repinning dependencies and running format scripts.
- Updated the renovate configuration to include specific commit message prefixes and labels for various package managers.
- Increased the `prConcurrentLimit` from 5 to 15 to allow more concurrent PRs.
- Modified existing CI workflows to exclude renovate branches from format and test jobs.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>ci-rbe.yml</strong><dd><code>Exclude renovate branches from CI format and test jobs</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.github/workflows/ci-rbe.yml

<li>Added condition to exclude renovate branches from format and test <br>jobs.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14709/files#diff-9ad6de1290c9f9260ba7c459ff1aa974c7c15aeeaae6f621aeda5dcde357b30e">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>ci-renovate-rbe.yml</strong><dd><code>Add CI workflow for renovate branches</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.github/workflows/ci-renovate-rbe.yml

<li>Added a new CI workflow specifically for renovate branches.<br> <li> Included steps for repinning dependencies and running format scripts.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14709/files#diff-3c36acad7162c66d69ced5efdd84650ec36e90e0c8fad0fccf42f4553851b623">+53/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>renovate.json</strong><dd><code>Update renovate configuration for package management</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

renovate.json

<li>Updated commit message prefixes and labels for various package <br>managers.<br> <li> Increased <code>prConcurrentLimit</code> from 5 to 15.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14709/files#diff-7b5c8955fc544a11b4b74eddb4115f9cc51c9cf162dbffa60d37eeed82a55a57">+62/-16</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by VietND96** [comment]
For renovate's PR, sometimes we have to run format manually in PR branch.
Sometimes in commits (e.g preparation for release), bulk of dependencies are updated via bazel run.
What is your view on renovate's PR? @diemol, @p0deje, @harsha509

**Comment by p0deje** [comment]
@VietND96 Ideally we'd wire up the format + dependencies update command to automatically run on Renovate PRs and commit to the same branch so that maintainers only need to review PR and merge it. If the whole automation just works, we can enable auto-merge for renovate PRs as long as CI passes.

**Comment by DineshKumarRA** [comment]
@VietND96 @p0deje I can create a separate workflow for Renovate PRs to pin the maven dependencies and commit using the Selenium CI bot. Do you also want to add format script at the end before the commit?

**Comment by DineshKumarRA** [comment]
@VietND96 @p0deje @diemol This is now ready for review

File: .github/workflows/ci-renovate-rbe.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            echo "code_changes=true" >> $GITHUB_ENV
+          fi
+      - name: Push changes
+        if: ${{ env.code_changes }} == 'true'
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.SELENIUM_CI_TOKEN }}
+          branch: ${{ github.head_ref }}
+      - name: Format
+        uses: ./.github/workflows/bazel.yml
```
Comment (p0deje) [inline .github/workflows/ci-renovate-rbe.yml]: I don't think you can use workflow from step.

**Review by p0deje — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-renovate-rbe.yml (hunk 1 above)
Comment (DineshKumarRA) [reply .github/workflows/ci-renovate-rbe.yml]: added it as a separate action which depends on fixing the formatting issue

**Comment by DineshKumarRA** [comment]
@p0deje @VietND96 There is "maven_install.json" and sometimes "Cargo.Bazel.lock" being generating when running the repin. Does both the files need to be committed into the branch?
If yes, then I can change the git add command to 'git add .' rather than "git add ./java/maven_install.json" to add all the changed files

**Comment by p0deje** [comment]
@DineshKumarRA Yes, we need to commit both maven_install.json and Cargo.Bazel.lock.

**Comment by DineshKumarRA** [comment]
@p0deje @VietND96 This is now ready. JS dependency is still failing due to https://github.com/aspect-build/rules_js/issues/1445. But others are all fixed now.

**Comment by DineshKumarRA** [comment]
@VietND96 @p0deje @diemol Have updated the repin dependencies for all languages. It is now ready for review.

**Pull request merged**